src/core/lib/surface/call.cc

```diff
@@ -553,10 +553,10 @@ static void destroy_call(void* call, grpc_error* error) {
                                             grpc_schedule_on_exec_ctx));
 }
 
-void grpc_call_ref(grpc_call* c) { gpr_ref(&c->ext_ref); }
+void grpc_call_ref(grpc_call* c) { c->ext_ref.Ref(); }
 
 void grpc_call_unref(grpc_call* c) {
-  if (!gpr_unref(&c->ext_ref)) return;
+  if (GPR_UNLIKELY(!c->ext_ref.Unref())) return;
```

Comment:
How is the likelihood determined? I guess it's actually likely that `c->ext_ref.Unref() == false`?

Reply:
Oh darn. Thanks for catching this . This is a typo. My brain said likely but my hand wrote unlikely. I'm away from desk, will fix tonight.

Reply:
This should be fixed now. Thank you for catching this again.